src/audio/Audio.js

```diff
@@ -78,13 +78,20 @@ Audio.prototype = Object.assign( Object.create( Object3D.prototype ), {
 
 		}
 
+		if ( when === undefined || when === 0 ) when = this.context.currentTime;
```

Comment:
So it will be basically impossible to call `source.start( 0 )`? I don't use the WebAudio API much, but I want to double check we're not missing user cases.